Dockerfile

```diff
@@ -49,36 +47,51 @@ RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then apt-get -y --no-install-r
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* ; fi
 
-# Update pip and install requirements
-COPY requirements.txt dev-requirements.txt mssql-requirements.txt ./
-RUN pip install -U pip  \
-    && pip --no-cache-dir install -r requirements.txt -r dev-requirements.txt \
-    && if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then pip --no-cache-dir install -r mssql-requirements.txt ; fi
+#########################
+## Python Dependencies ##
+#########################
+COPY mssql-requirements.txt .
+RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then pip --no-cache-dir install -r mssql-requirements.txt ; fi
+
+COPY dev-requirements.txt .
+RUN pip install -U pip --no-cache-dir install -r dev-requirements.txt
 
-# Copy in the application files and install it locally
+COPY requirements.txt .
+RUN pip install -U pip --no-cache-dir install -r requirements.txt
+
+###############################
+## General Application Setup ##
+###############################
 COPY . /fidesops
 WORKDIR /fidesops
-RUN pip install -e .
+
+# Immediately flush to stdout, globally
+ENV PYTHONUNBUFFERED=TRUE
 
 # Enable detection of running within Docker
 ENV RUNNING_IN_DOCKER=true
 
-############
-## Worker ##
-############
-FROM backend as worker
-CMD [ "fidesops", "worker" ]
-
-#################
-## Application ##
-#################
-## Set the image up to be the application
-FROM backend as app
-
 # Make a static files directory
 RUN mkdir -p /fidesops/src/fidesops/build/static/
 
+EXPOSE 8080
+CMD [ "fidesops", "webserver" ]
+
+#############################
+## Development Application ##
+#############################
+FROM backend as dev
+
+RUN pip install -e .
+
+#############################
+## Production Application ##
+#############################
+FROM backend as prod
+
+# Install without a symlink
+RUN python setup.py sdist
+RUN pip install dist/fidesops-*.tar.gz
```

Comment:
this is so that we install our package the same way that a user using `pip` would. It enables us to test our `prod` docker image and know that we'd get the same results as if we did a local `pip` install and tested it that way.

doing `pip install -e .` covers up a lot of path issues (for instance, fidesops was missing a few `__init__.py` files but we didn't know until we tried to publish to pypi)

Reply:
Ok, let me see if a clean build resolves this error also.